runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorker.java

```diff
@@ -174,7 +174,7 @@ private StreamingDataflowWorker(
       StreamingCounters streamingCounters,
       MemoryMonitor memoryMonitor,
       GrpcWindmillStreamFactory windmillStreamFactory,
-      Function<String, ScheduledExecutorService> executorSupplier,
+      Function<String, ScheduledExecutorService> activeWorkRefreshExecutorFn,
```

Comment:
does this need to be a parameter still? can we just create a scheduled executor to pass into ActiveWorkRefresher?

Reply:
done